analysed-packages/jquery/version-3.4.1/jquery-3.4.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,4203 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_jquery-3.4.1.tar.gz_1717573799.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of jquery 3.4.1.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-06-05T07:50:00Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: jquery-3.4.1.tar.gz
+PackageFileName: jquery-3.4.1.tar.gz
+SPDXID: SPDXRef-upload153
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: 32e1f99f2b69c5669c0777ad564950eca8b22f4b
+PackageChecksum: SHA1: a13b629fef50b0a18b450a75b364d76e7fd0782a
+PackageChecksum: SHA256: 4b58238a27515e242c3fca7f56b4b9d99229cf40f6d6f84554f55ce9d2633c19
+PackageChecksum: MD5: 35184010253a13f3512b7995a7a4d51d
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-MIT
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
```

Comment:
In the disclosure file you have listed the contributors but not here, why? can you please add them here too:

> PackageCopyrightText: <text> Authors ordered by first contribution. .... </text>